components/ieee802154/driver/esp_ieee802154_dev.c

```diff
@@ -477,7 +475,7 @@ static IRAM_ATTR void isr_handle_rx_abort(void)
     case IEEE802154_RX_ABORT_BY_TX_ACK_STOP:
     case IEEE802154_RX_ABORT_BY_ED_STOP:
         // do nothing
-        break;
+        return;
```

Comment:
Here we need to set next operation because of the feature `pending tx`. If we want to send a packet while the HW is receiving a packet. We will enable all rx abort at this time and pending the transmition. If any aborts arrived to HW. We should stop the Rx and transmitting the pending packet immidiately.

Reply:
And for your scenario: RX stop, the ISR IEEE802154_RX_ABORT_BY_RX_STOP should be disabled for almost time(only will be enable when there is a TX pending). So only risk is when stop RX while there is a pending tx.